docs(openapi): update setup options for fastify (#1983)
nestjs/docs.nestjs.com: content/openapi/introduction.md

#### Setup options

> You can configure Swagger UI by passing the options object which fulfills the `ExpressSwaggerCustomOptions` interface as a fourth argument of the `SwaggerModule#setup` method, if you use express.

Reviewer: ```suggestion
You can configure Swagger UI by passing the options object which fulfills the `ExpressSwaggerCustomOptions` (if you use express) interface as a fourth argument of the `SwaggerModule#setup` method.
```

}
```

> while if you use fastify you can configure the user interface using `FastifySwaggerCustomOptions`.

Reviewer: ```suggestion
If you use fastify, you can configure the user interface by passing the `FastifySwaggerCustomOptions` object.
```
